core/tests/karma-globals.js

```diff
@@ -81,7 +81,8 @@ var GLOBALS = {
     id: 'en',
     text: 'English'
   }],
-  ASSET_DIR_PREFIX: ''
+  ASSET_DIR_PREFIX: '',
+  TEMPLATE_DIR_PREFIX: 'core/templates/dev/head'
```

Comment:
It's needed for the directives to be fetched correctly, otherwise Karma doesn't know where to get them from. Good question though, I will add a comment.